oar/core/util.py

```diff
@@ -1,5 +1,5 @@
 import logging
-
+import semver
```

Comment:
@rioliu-rh do we need to add this lib into the dependency, since I doubt semver is a standard python lib.

Reply:
it is already included in art tools

Reply:
/lgtm